### ISSUE TITLE
Runtime error

### ISSUE THREAD

**Issue by devnixio** [loftylabs/django-developer-panel#2]
Installed middleware and plugin per directions.

When I run my server I open localhost and all seems fine except I can't really access the plugin.  Options is grayed out and no access to actual panel.  I refresh and receive:

```
Request Method:	GET
Request URL:	http://localhost:8000/
Django Version:	1.11.1
Exception Type:	RuntimeError
Exception Value:	
dictionary changed size during iteration 
```

Traceback

```
Environment:


Request Method: GET
Request URL: http://localhost:8000/

Django Version: 1.11.1
Python Version: 3.6.0
Installed Applications:
['django.contrib.admin',
 'django.contrib.auth',
 'django.contrib.contenttypes',
 'django.contrib.sessions',
 'django.contrib.messages',
 'django.contrib.staticfiles',
 'base',
 'blog',
 'tinymce']
Installed Middleware:
['djdev_panel.middleware.DebugMiddleware',
 'django.middleware.security.SecurityMiddleware',
 'django.contrib.sessions.middleware.SessionMiddleware',
 'django.middleware.common.CommonMiddleware',
 'django.middleware.csrf.CsrfViewMiddleware',
 'django.contrib.auth.middleware.AuthenticationMiddleware',
 'django.contrib.messages.middleware.MessageMiddleware',
 'django.middleware.clickjacking.XFrameOptionsMiddleware']



Traceback:

File "C:\Users\DCS\AppData\Local\Programs\Python\Python36-32\lib\site-packages\django\core\handlers\exception.py" in inner
  41.             response = get_response(request)

File "C:\Users\DCS\AppData\Local\Programs\Python\Python36-32\lib\site-packages\django\core\handlers\base.py" in _get_response
  178.             response = middleware_method(request, callback, callback_args, callback_kwargs)

File "C:\Users\DCS\AppData\Local\Programs\Python\Python36-32\lib\site-packages\djdev_panel\middleware.py" in process_view
  143.         for key in VIEW_METHOD_DATA.keys():

Exception Type: RuntimeError at /
Exception Value: dictionary changed size during iteration

```

**Comment by ckinsey** [comment]
@devnixio I was able to reproduce this under Python 3 and have opened a pull request for the fix.  Can you confirm the fix against your project by installing the `gh-issue-2` branch?

**Comment by primoz-k** [comment]
@ckinsey I've had the same issue and `gh-issue-2` seems to fix it.

**Comment by ckinsey** [comment]
This issue was fixed in django-developer-panel-0.1.2, please update your copy!
